modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -22,3 +22,53 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 .. Select the *openshift-gitops-cluster* instance to display the password.
 .. Copy the password.
 . Use this password and `admin` as the username to log in to the Argo CD UI in the new window.
+
+The default Argo CD instance installed by the OpenShift GitOps Operator as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
+. Label the existing nodes:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
++
+.Optional: If required, you can also apply taints and isolate the workloads on infrastructure nodes and prevent other workloads from scheduling on these node:
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add `runOnInfra` toggle in the `GitOpsService` custom resource:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Add `tolerations` to the `GitOpsService` custom resource if taints have been added to the nodes, for example:
```

Comment:
Since adding taints is optional, I think this step is optional too. @reginapizza can you confirm?
Also, suggest starting the sentence with - If taints have been added to the nodes, then add `tolerations`...
This helps users to decide if it is worth reading the next part of the sentence. They will read it only if they have used taints.
Also, in short can we mention what these tolerations do like how taints are used to isolate workloads?

Reply:
@reginapizza please suggest

Reply:
@saumeya can you provide input? I believe this step is optional as well but I'm not positive

Reply:
@Preeticp yes that step is optional, we can add your suggestion, i'll try adding a brief about tolerations and taints, there is a jira about already assigned to me, i'll work on it.